packages/flutter/lib/src/rendering/object.dart

```diff
@@ -453,6 +453,30 @@ class PaintingContext extends ClipContext {
     return PaintingContext(childLayer, bounds);
   }
 
+  void _checkForClipBehavior(String type) {
```

Comment:
nit: `_debugCheckForClipBehavior`

Reply:
Alternatively, remove the assert from this method and make it a `bool Function()` that could be passed directly to `assert` where it is called

Reply:
Done

Reply:
or I guess, more like a `bool Function() Function(String)`